bridge/test/nftbridge/NFTBridge_test.js

```diff
@@ -190,6 +200,126 @@ contract("Bridge NFT", async function(accounts) {
           );
         });
       });
+
+      describe("Fixed Fee", async function() {
+        it("Federator should receive fee", async function() {

```

Comment:
I'd maybe rename to `"should send fee to federator correctly"` so it reads better with the `it` call

Reply:
done at https://github.com/rsksmart/tokenbridge/pull/248/commits/5e908cbcb56c8924846eff80da10efa0adef6d0f